cellpack/autopack/upy/simularium/simularium_helper.py

```diff
@@ -488,15 +490,17 @@ def init_scene_with_objects(
         if grid_point_positions is not None:
 
             for index in range(len(grid_point_compartment_ids)):
-                if index % 10 == 0:
+                if index % 1 == 0:
```

Comment:
Isn't `n % 1` always `== 0`?

Reply:
yeah, I use this to modulate the amount of grid points based on how slow it is to load in Simularium. I could try to do this more computationally, but I've just been changing it as needed when I'm debugging something